### ISSUE TITLE
benchmark

### ISSUE THREAD

**Issue by kylak** [uNetworking/uWebSockets#1737]
Could it be possible to have in the benchmark, the Rust crate [web-socket](https://github.com/nurmohammed840/websocket.rs) ? since web-socket seems to be the faster for Rust ? (keeping also the ws Rust crate would still be interesting).

**Comment by uNetworkingAB** [comment]
Yes that can be a good idea whenever the benchmark needs updating but right now, this is not a priority. For version 21 it can be relevant again
